### PR TITLE
Adding browser support page.

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by pjackson28** [comment]
A French page is needed as well, otherwise we'll have a 404 when the language link is used.

**Comment by LaurentGoderre** [comment]
Actually, the build would fail because of the broken link

**Comment by pjackson28** [comment]
Not sure why the build isn't broken but we shouldn't be providing an English page without an equivalent French page. It would even be better to have the French page with the English text (using the appropriate lang attribute) rather than no French page at all (although it should still be translated).

File: site/pages/docs/browser-support-en.hbs [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+{
+	"title": "Browser Support",
+	"language": "en",
+	"description": "Browser support for the Web Experience Toolkit.",
+	"altLangPrefix": "index",
```
Comment (LaurentGoderre) [inline site/pages/docs/browser-support-en.hbs]: This is the reason why it's not failing. It's pointing to another page which might actually be worse. I would recommend copying the english page, wrapping in a `<div lang="en"></div>` in the meantime and change this value to `browser-support`

**Comment by pjackson28** [comment]
Can the missing language page and incorrect language link be fixed so this can be merged in? Also the message should be changed to "Documentation: Added browser support page"

**Comment by shawnthompson** [comment]
Can we finish this? and would this be the "official" list?

**Comment by andre-gh** [comment]
Can this pull request be closed, has it gone stale?

**Comment by andre-gh** [comment]
Due to going stale this pull request has been closed.  Please feel free to re-open and comment should you wish to continue development.

**Pull request closed without merging**